### PR TITLE
(PE-33493) Add Ubuntu 2204 codename

### PR DESCRIPTION
**Pull request by cthorn42**

### CODE REVIEW

File: lib/beaker/platform.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@ class Platform < String
                      "wheezy"    => "7",
                      "squeeze"   => "6",
                    },
-        :ubuntu => { "focal"   => "2004",
+        :ubuntu => { "jazzy"   => "2204",
```
Comment (joshcooper) [inline lib/beaker/platform.rb:16]: I think 22.04 is codenamed "jammy"

Comment (cthorn42) [reply lib/beaker/platform.rb]: Ugh, late night PR, what could go wrong? Thanks for catching this.

File: spec/beaker/platform_spec.rb [hunk 2]
```diff
@@ -82,6 +82,11 @@ module Beaker
         expect( platform.with_version_codename ).to be === 'debian-squeeze-xxx'
       end
 
+      it "can convert unbuntu-2204-xxx to ubuntu-jazzy-xxx" do
```
Comment (joshcooper) [inline spec/beaker/platform_spec.rb:85]:
```suggestion
      it "can convert unbuntu-2204-xxx to ubuntu-jammy-xxx" do
```

Comment (kenyon) [inline spec/beaker/platform_spec.rb:85]:
```suggestion
      it "can convert ubuntu-2204-xxx to ubuntu-jammy-xxx" do
```

it's misspelled "unbuntu" three times in this file

File: spec/beaker/platform_spec.rb [hunk 3]
```diff
@@ -82,6 +82,11 @@ module Beaker
...
+        @name = 'ubuntu-2204-xxx'
+	expect( platform.with_version_codename ).to be === 'ubuntu-jammy-xxx'
+      end
+
       it "can convert unbuntu-2004-xxx to ubuntu-focal-xxx" do
```
Comment (kenyon) [inline spec/beaker/platform_spec.rb:90]:
```suggestion
      it "can convert ubuntu-2004-xxx to ubuntu-focal-xxx" do
```

**Review by kenyon — CHANGES_REQUESTED** [review]

File: spec/beaker/platform_spec.rb (hunk 2 above)
Comment (cthorn42) [reply spec/beaker/platform_spec.rb]: 👍 fixed up

**Review by joshcooper — APPROVED** [review]

**Pull request merged**